BinaryObjectScanner/Protection/SecuROM.cs

```diff
@@ -108,16 +106,13 @@ public class SecuROM : IExecutableCheck<PortableExecutable>, IPathCheck
 
             // Check if executable contains a SecuROM Matroschka Package
 
-            if (exe.MatroschkaPackage != null)
-            {
-                var package = exe.MatroschkaPackage;
-                if (package == null)
-                    return null;
-                
-                var matroschka = CheckMatroschkaPackage(package, includeDebug);
-                if (matroschka != null)
-                    return matroschka;
-            }
+            var package = exe.MatroschkaPackage;
+            if (package == null)
+                return null;
```

Comment:
You probably don't want to return null here. No other checks at all would run. I think what you meant to do is invert the null check and put the `CheckMatroschkaPackage` back inside, similar to the original implementation.